tests/llmcompressor/transformers/obcq/test_consecutive_runs.py

```diff
@@ -5,14 +5,19 @@
 import pytest
 import yaml
 from parameterized import parameterized_class
+from transformers import AutoModelForCausalLM
+from transformers.utils.quantization_config import CompressedTensorsConfig
 
+from llmcompressor.transformers.utils.helpers import resolve_recipe
 from tests.testing_utils import parse_params, requires_gpu
 
 CONFIGS_DIRECTORY = "tests/llmcompressor/transformers/obcq/obcq_configs/consec_runs"
 GPU_CONFIGS_DIRECTORY = (
     "tests/llmcompressor/transformers/obcq/obcq_configs/consec_runs/gpu"
 )
 
+quantization_config = CompressedTensorsConfig(run_compressed=False)
```

Comment:
nit: consider moving into class, similar to other tests

Reply:
ok